build/node.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
-# based on https://github.com/joyent/docker-node/blob/787d816f90dc1bf933b6503b876bdeb752b24f96/0.10/Dockerfile
+# based on https://github.com/nodejs/docker-node/blob/787d816f90dc1bf933b6503b876bdeb752b24f96/0.12/Dockerfile
 
+# verify gpg and sha256: http://nodejs.org/dist/v0.10.30/SHASUMS256.txt.asc
```

Comment:
It is a copy-paste from the official node Dockerfile: https://github.com/nodejs/docker-node/blob/787d816f90dc1bf933b6503b876bdeb752b24f96/0.12/Dockerfile#L3 with just the ``RUN`` statements removed.